scipy/spatial/qhull.pyx

```diff
@@ -2410,3 +2449,269 @@ class Voronoi(_QhullUser):
+                    For other dimensions, they are in input order.
+    dual_simplices : ndarray of ints, shape (nfacet, ndim)
+                     Indices of points forming the simplical facets of the dual convex hull.
+    dual_neighbors : ndarray of ints, shape (nfacet, ndim)
+                     Indices of neighbor facets for each facet.
+                     The kth neighbor is opposite to the kth vertex.
+                     -1 denotes no neighbor.
+    dual_equations : ndarray of double, shape (nfacet, ndim+1)
+                     [normal, offset] forming the hyperplane equation of the dual facet
+                     (see `Qhull documentation <http://www.qhull.org/>`__  for more).
+    coplanar : ndarray of int, shape (ncoplanar, 3)
+        Indices of coplanar points and the corresponding indices of
+        the nearest facets and nearest vertex indices.  Coplanar
+        points are input points which were *not* included in the
+        triangulation due to numerical precision issues.
+
+        If option "Qc" is not specified, this list is not computed.
+    dual_area : float
+        Area of the dual convex hull
+    dual_volume : float
+        Volume of the dual convex hull
+
+    Raises
+    ------
+    QhullError
+        Raised when Qhull encounters an error condition, such as
+        geometrical degeneracy when options to resolve are not enabled.
+    ValueError
+        Raised if an incompatible array is given as input.
+
+    Notes
+    -----
+    The intersections are computed using the 
+    `Qhull library <http://www.qhull.org/>`__.
+
+    Examples
+    --------
+
+    TODO: example
+    Halfspace intersection of planes forming some polygon
+
+    >>> from scipy.spatial import HalfspaceIntersection
+    >>> import numpy as np
+    >>> halfspaces = np.array([[-1, 0., 0.],
+    >>>                        [0., -1., 0.],
+    >>>                        [2., 1., -4.],
+    >>>                        [-0.5, 1., -2.]])
+    >>> feasible_point = np.array([0.5, 0.5])
+    >>> hs = HalfspaceIntersection(halfspaces, feasible_point)
+
+    Plot halsfpaces as filled regions and intersection points:
+
+    >>> import matplotlib.pyplot as plt
+    >>> xlim, ylim = (-1, 3), (-2, 6)
+    >>> x = np.linspace(-1, 3, 100)
+    >>> symbols = ['-', '+', 'x', '*']
+    >>> signs = [0, 0, -1, -1]
+    >>> for h, sym, sign in zip(halfspaces, symbols, signs):
+    >>>   if h.item(1) == 0:
+    >>>     plt.axvline(-h.item(2)/h.item(0), label='{}x+{}y+{}=0'.format(*h.tolist()))
+    >>>     xi = np.linspace(xlim[sign], -h.item(2)/h.item(0), 100)
+    >>>     plt.fill_between(xi, ylim[0], ylim[1], color="none", edgecolor="b", hatch=sym, alpha=0.5)
+    >>>   else:
+    >>>     plt.plot(x, (-h.item(2)-h.item(0)*x)/h.item(1), label='{}x+{}y+{}=0'.format(*h.tolist()))
+    >>>     plt.fill_between(x, (-h.item(2)-h.item(0)*x)/h.item(1), ylim[sign], color="none", edgecolor="b", hatch=sym, alpha=0.5)
```

Comment:
What I meant here is to limit the line length to 80 chars or so --- 
there's no line wrapping in the code examples and it's annoying if you 
have to scroll horizontally to see the code.